spyderlib/spyder.py

```diff
@@ -1718,17 +1669,17 @@ def setup_default_layouts(self, index, settings):
         self._custom_layout_timer.start(100)

```

Comment:
I did it already, there is no difference. Only when I comment it, I can prevent the overflowing :-)